### ISSUE TITLE
initialize crashes if pr_ordata is not installed

### ISSUE THREAD

**Issue by mkoval** [personalrobotics/herbpy#52]
I believe this is related to the recent changes to the perception pipeline we merged in.

@Shushman @jeking04 Can you take a look?

```
Traceback (most recent call last):
  File "/home/mkoval/storage/herb-ws/src/herbpy/scripts/console.py", line 47, in <module>
    env, robot = herbpy.initialize(**herbpy_args)
  File "/home/mkoval/storage/herb-ws/src/herbpy/src/herbpy/herb.py", line 87, in initialize
    prpy.bind_subclass(robot, HERBRobot, **kw_args)
  File "/home/mkoval/storage/herb-ws/src/prpy/src/prpy/bind.py", line 366, in bind_subclass
    instance.__init__(*args, **kw_args)
  File "/home/mkoval/storage/herb-ws/src/herbpy/src/herbpy/herbrobot.py", line 188, in __init__
    'data/objects')
  File "/home/mkoval/storage/herb-ws/src/prpy/src/prpy/util.py", line 680, in FindCatkinResource
    relative_path))
IOError: Loading resource "data/objects" failed.
```

**Comment by jeking04** [comment]
Whoops. Yup that was from adding the pipeline. I'll add logic to warn and not load the perception module if pr-ordata isn't found, but not fail.

**Comment by mkoval** [comment]
Fixed in #53.
